.coveragerc

```diff
@@ -1,3 +1,7 @@
 [report]
-include =
```

Comment:
@hvelarde could you elaborate why this didn't work? An omit statement is harder to maintain if people add stuff. An include is a better solution in my option (only if it works of course).